docs/modules/ROOT/partials/install/prepare-terraform.adoc

```diff
@@ -1,17 +1,3 @@
-
```

Comment:
This needs to stay in the partial, or the Exoscale install instructions need to be updated accordingly.